### ISSUE TITLE
Additional Software PINNs/NOs

### ISSUE THREAD

**Issue by dario-coscia** [bitzhangcy/Neural-PDE-Solver#1]
Hello👋🏻 
First of all, thank you for the great repository you made, is an invaluable tool for reference for researchers in academia and industry!

I saw an extensive software section, and I suggest you look at [PINA](https://github.com/mathLab/PINA) if you want to add another one! PINA is an open-source Python library providing an intuitive interface for solving differential equations using PINNs, NOs or both together based on [PyTorch](https://pytorch.org/) and [PyTorchLightning](https://lightning.ai/docs/pytorch/stable/). In addition to implementing the state-of-the-art methodology, we provide advancement training strategies such as multiple device training, modern model compression techniques, gradient accumulation, and so on thanks to `PyTorchLightning`.

If you find the software interesting and helpful it would be amazing to enter the list! 😄🚀

Paper Link: https://joss.theoj.org/papers/10.21105/joss.05352

**Comment by DK-fish** [comment]
收到，感谢你（们）的祝福！

**Comment by bitzhangcy** [comment]
Thank you for your suggestion. I've incorporated the open-source PINA library into my repository, confident it will benefit many new researchers in this field.

**Comment by DK-fish** [comment]
收到，感谢你（们）的祝福！
